HTML Forms Project

GzimAsani opened the pull request:
Project completed and apply for checkup.
In this project we:
We created a main element to contain all the logos and the main section
We used the form element to group inputs and labels
We used font awesome to create logos inside the buttons
We created two footer elements one together with the elements inside the main and the other outside  of it.

GzimAsani committed:
Adding linters

Rayan84 committed:
Added HTML and CSS Content

Added HTML and CSS Content

GzimAsani committed:
Adding Intuit logo, Mint, QuickBooks and turbo

Rayan84 committed:
Added form

Added form

GzimAsani committed:
Improved the main and also added the footer

Rayan84 committed:
Style fixes

font
removed borders

GzimAsani committed:
Improved the first and second footer

Rayan84 committed:
Position and style changes

Moved labels to the left
Increased inputs width
Placed lock icon into the password input
Border style changes

GzimAsani committed:
Improved the second footer and added the privacy settings

GzimAsani committed:
Adding linters

GzimAsani committed:
Improved the README file

GzimAsani committed:
Improved the README file

GzimAsani committed:
Improvements according to linters

GzimAsani committed:
Improving based on the linters

GzimAsani committed:
Some more improvements according to linters

GzimAsani committed:
CSS updates for linters

GzimAsani committed:
Linters

GzimAsani committed:
Improving the footer

GzimAsani committed:
New changes on CSS to fix the linters

GzimAsani committed:
Some more fixing

krys2fa commented on index.html:
```diff
@@ -0,0 +1,96 @@
+          </div>
+
+          <div class = 'create-account'>
+            <button id="creation"><i class="fas fas2 fa-unlock-alt create"></i>Create Account</button>
```
- [ ] [REQUIRED] The button should have a type of `submit` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/)).

krys2fa reviewed (changes requested):
## Nicely done @GzimAsani and @Rayan84 👏 
Your project displays good use of semantic HTML as well as CSS classes. 💪 
You have a professional looking README file as well. 👍 

Just a few more adjustments to complete the project. They can be seen below:
### Status: Changes Required ♻️ 

**NB:** Review was done at a screen size of **1440px**. Kindly state your preferred screen size if your clone displays better on your machine.

- [x] [REQUIRED] Your page does not fit on the screen hence there is a horizontal scroll-bar at the bottom of the page. Please fix this.

- [x] [REQUIRED] The lock icon for the password input field should be in the input field as seen below.

Original Page          |  Your Page
:-------------------------:|:-------------------------:
![original](https://user-images.githubusercontent.com/18187039/90802046-382cf200-e306-11ea-9b29-34e749123033.png) |  ![yours](https://user-images.githubusercontent.com/18187039/90802274-85a95f00-e306-11ea-859e-0630fe3da35c.png)

- [x] [REQUIRED] The copyright text below the form should be positioned at the center of the page (just like the form). The text in your page seems to be shifted a little more to the right. The comparison below with the original demonstrates this.

Original Page          |  Your Page
:-------------------------:|:-------------------------:
![original](https://user-images.githubusercontent.com/18187039/90802934-8393d000-e307-11ea-8ea9-046e4ffe8393.png) |  ![yours](https://user-images.githubusercontent.com/18187039/90802916-7aa2fe80-e307-11ea-96fd-316657ee208d.png)


Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

Rayan84 committed:
Reviewers Notes

Links color changed to match the source
Fixed lock icon position
Fixed  Legal, privacy, and security position
Removed excessive marign at the bottom

Rayan84 committed:
Punctuation fixes on the CSS

Punctuation fixes on the CSS

Rayan84 committed:
Removed excessive indentation

Removed excessive indentation

SpaYco reviewed (approved):
# Project Approved ✔️

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_
